migrate.go

```diff
@@ -87,6 +87,7 @@ func (b byId) Less(i, j int) bool { return b[i].Less(b[j]) }
 type MigrationRecord struct {
 	Id        string    `db:"id"`
 	AppliedAt time.Time `db:"applied_at"`
+	DownSql   string    `db:"down_sql"`
```

Comment:
This will be a problem, anyone who is currently using sql-migrate won't have this column in their migrations table.

We've come at the sad point that we'll need to migrate the migrations table, but we have no way of doing so.

Reply:
Hmmm. What if we store the `DownSql` in a separate new table? Does that let us avoid compatibility issues?